NEWS.md

```diff
@@ -1,3 +1,10 @@
+# New in snapd 2.60.2:
+* Bugfixes
+* Performance improvements for apparmor_parser to compensate for
+  the slower `-O expr-simplify` default used. This should bring
+  the performance back to the 2.60 level and even increase it
+  for many use-cases.
```

Comment:
nitpick: maybe the order of the two should be swapped

Reply:
Excellent point, I updated it